apps/envited.ascs.digital/common/validator/shacl/shacl.ts

```diff
@@ -164,12 +178,19 @@ export const validateDomainMetadata = _validateDomainMetadata({
 })
 
 export const _checkIfAllFilesInManifestExists =
-  ({ getShaclDataFromZip }: { getShaclDataFromZip: (file: File, fileName: string) => Promise<string> }) =>
+  ({ getShaclDataFromZip }: { getShaclDataFromZip: (file: File, fileName: string) => Promise<any> }) =>
```

Comment:
We need a correct return type here

Reply:
I forgot to put it back as it was, when I was fixing the error messages